com.unity.netcode.gameobjects/Editor/NetworkBehaviourEditor.cs

```diff
@@ -37,8 +37,8 @@ private void Init(MonoScript script)
                 var ft = fields[i].FieldType;
                 if (ft.IsGenericType && ft.GetGenericTypeDefinition() == typeof(NetworkVariable<>) && !fields[i].IsDefined(typeof(HideInInspector), true))
                 {
-                    m_NetworkVariableNames.Add(fields[i].Name);
-                    m_NetworkVariableFields.Add(fields[i].Name, fields[i]);
+                    m_NetworkVariableNames.Add(ObjectNames.NicifyVariableName(fields[i].Name));
```

Comment:
really? we have this public API? lol
I'd call it "Prettify" or "Beautify" as seen in the web/backend industry but OK :P